codegen/package.json

```diff
@@ -25,11 +25,13 @@
   },
   "homepage": "https://github.com/xataio/client-ts/blob/main/codegen/README.md",
   "dependencies": {
+    "@xata.io/client": "0.0.0-alpha.3bb1ec0",
```

Comment:
Probably a peerDependency here?

Reply:
🤔 Why do you prefer to install it as a `peerDependency`?